### PR TITLE
RichPresence Add-ons

### PR DESCRIPTION
**Pull request by Bastiti**
It might be useless for some people.
But, I'm getting bored.

I added ithis inside the boot folder cause it's the most accurate folder.
No need to tell more. 
Useless PR, but not useless for some people.

Comments Added.

### CODE REVIEW

File: boot/client/main.lua [hunk 1]
```diff
@@ -34,3 +34,14 @@ if Config.EnableHud then
   end)
 
 end
+
+--RichPresence
+if Config.EnableRichPresence then
+  local playerId = PlayerId()
+   SetDiscordAppId(tonumber(GetConvar("RichAppId", "757218164345012224")))
```
Comment (itschip) [inline boot/client/main.lua:41]: Is those numbers your own Discord App Id? Just wondering. Maybe remove it and tell add a comment that links to the developers portal for Discord.

Comment (Bastiti) [reply boot/client/main.lua]: Done :p

Actual view.
image

Comment (itschip) [reply boot/client/main.lua]: 👌

**Comment by ArkSeyonet** [comment]
Pushed this PR and fixed any conflicts that existed in another PR for you without waiting to ask you to update the PR.

**Pull request closed without merging**